### PR TITLE
Docs - Add tables and header fixes to primary readme

### PR DESCRIPTION
**Pull request by lee5378**
- Converted block/reward information to more organized tables in primary readme for readability
- Fixed sub-headers under "development tips and tricks" in primary readme for readability

### CODE REVIEW

**Comment by AbcSxyZ** [comment]
Thx for your PR. To let you know, there is some change on the way for the [README.md](https://github.com/dogecoin/dogecoin/pull/2313). It's in progress, your suggested change probably need to be adapted depending on merged changes, block reward may be moved somewhere else.

If you want to contribute to the documentation or see some of the ongoing work on it, take a look at [Docs4Doge](https://github.com/dogecoin/dogecoin/issues/2271) :)

File: README.md [hunk 1]
```diff
@@ -81,37 +81,28 @@ lessen the impact of sudden increases and decreases of network hashing rate.
...
-200,000–299,999: 125,000 Dogecoin
-
-300,000–399,999: 62,500 Dogecoin
-
-400,000–499,999: 31,250 Dogecoin
-
-500,000–599,999: 15,625 Dogecoin
-
-600,000+: 10,000 Dogecoin
+|Block |Reward
```
Comment (patricklodder) [inline README.md:84]: Could you make the tables non-condensed? Then it's pretty in html render and as plaintext

**Review by patricklodder — CHANGES_REQUESTED** [review]
Would be great if tables were non-condensed, ACK on everything else.

File: README.md (hunk 1 above)
Comment (lee5378) [reply README.md:84]: Could you link me to an example of a non-condensed table?

Comment (patricklodder) [reply README.md:84]:
```
| Block          | Reward        |
| :------------- | ------------: |
| 0-100k         |     1,000,000 |
| 100k-200k      |       500,000 |
```

Comment (lee5378) [reply README.md:84]: Oh I see, I'll get that updated.

Comment (lee5378) [reply README.md]: @patricklodder Revision applied!

**Comment by lee5378** [comment]
Rebased to 1.14.4-dev, latest PR is #2395

**Pull request closed without merging**